openedx/openedx-wordpress-ecommerce
# docs: update the readmes and the how-tos, add quickstart and a tutorial

## Description

This PR:
- Fix the docs (The decision 4 wasn't in any index)
- Update the Quickstart, the How-tos, and the READMEs
- Create a Tutorial

## How to test
- Review the doc files here: https://github.com/eduNEXT/openedx-wordpress-ecommerce/tree/mfmz/update-doc/docs
- Or you can run an environment to see the final result of the doc. (I added screenshots)

### Run an environment
- Create a python8 venv and activate it
```
virtualenv -p python3.8 venv
source venv/bin/activate
```
- Install requeriments
```
make requirements
```
- Enter in docs
```
cd docs
```
- Compile
```
make html
```
- Run the server
```
cd ..
make serve_docs
```
(By default uses port 8000)
or
```
cd ..
python -m http.server 8200 --directory build/html
```
(If you need to serve this in another port)


## Screenshots
![Screenshot from 2023-11-10 12-49-03](https://github.com/openedx/openedx-wordpress-ecommerce/assets/35668326/7ff15189-da65-4f11-9cc5-97d8bb950804)
![Screenshot from 2023-11-10 12-49-45](https://github.com/openedx/openedx-wordpress-ecommerce/assets/35668326/26ec60eb-0efb-4bbc-953d-a8c78a2b04d3)
![Screenshot from 2023-11-10 12-50-37](https://github.com/openedx/openedx-wordpress-ecommerce/assets/35668326/48f00177-718d-4bb8-b0c9-116f1d3c1404)
